roadmap.md

```diff
@@ -72,6 +72,13 @@ On the runtime level, the streaming operators are extended to also support the d
 patterns required for some batch operations ([discussion thread](https://lists.apache.org/thread.html/cb1633d10d17b0c639c3d59b2283e9e01ecda3e54ba860073c124878@%3Cdev.flink.apache.org%3E)).
 This is also groundwork for features like efficient [side inputs](https://cwiki.apache.org/confluence/display/FLINK/FLIP-17+Side+Inputs+for+DataStream+API).
 
+# Multi-language support
+
+At the Flink's API level, we have already support DataStream API/DataSet API/Table API&SQL, and both DataSet API and 
+DataStream API already provides python API(flink-python and flink-stream-python, but they may be deprecated later). 
+To cater to the current large number of analytics users, we plan to redesign the Flink Python API and support 
```

Comment:
I would not call this a "redesign" of the Python API but a new Python Table API.